### PR TITLE
Allow users to generate thoth configuration during the build process

### PR DESCRIPTION
**Pull request by fridex**
## Related Issues and Dependencies

Related: https://github.com/thoth-station/s2i-thoth/issues/189

## This introduces a breaking change

- [x] No

### CODE REVIEW

**Comment by fridex** [comment]
I will open the flag for disabling all the thoth in a separate PR.

File: README.rst [hunk 1]
```diff
@@ -47,6 +47,7 @@ Configuration options for Thoth's s2i image:
 
 * ``THOTH_ADVISE`` - always use the recommended stack by Thoth (even if the lock file is present in the repo)
 * ``THOTH_CONFIG_CHECK`` - verify values stated in the configuration file match the build environment
+* ``THOTH_GENERATE_CONFIG`` - generate configuration file during the build process, overwrites already existing configuration file if present
```
Comment (goern) [inline README.rst:50]: this feels more like `THOTH_FORCE_GENERATION_OF_CONFIG` shall we make the overwrite more explicit to the user?

**Comment by goern** [comment]
What is the status of this?

**Review by KPostOffice — APPROVED** [review]

**Pull request merged**